cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/system/SystemWorkerServiceLauncher.java

```diff
@@ -165,6 +169,17 @@ public void run() {
                                                                                               secretPath));
           }
 
+          // Overwrite JVM options if specified
+          Set<String> runnables = Stream.of(SystemWorkerTwillRunnable.class.getSimpleName())
```

Comment:
@masoud-io Please clarify whether there are other dependent pods in the system worker we also need to add here. Thanks!

Reply:
Unlike task worker pods, system worker pods have only one container.